main.go

```diff
@@ -19,6 +19,7 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/redhat-appstudio/build-service/pkg/metrics"
```

Comment:
I'd name it build service metrics or something, otherwise it can be confused with k8s metrics